pkg/noderesourcetopology/least_numa.go

```diff
@@ -230,3 +233,14 @@ func checkResourcesFit(identifier string, qos v1.PodQOSClass, resources v1.Resou
 
 	return true
 }
+
+func isInvalidCombineResources(numaNodes NUMANodeList, resources v1.ResourceList, combination []int) bool {
```

Comment:
REALLY minor, only if you resubmit: I'd reorder the args as
```
func isInvalidCombineResources(numaNodes NUMANodeList, combination []int, resources v1.ResourceList) bool
```
to be (more) consistent with `combineResources`

Reply:
Your suggestion is very good, I'll make a change

Reply:
thanks for that. I'll give time to other reviewers to have a look and LGTM when they are happy. Approval already done.